tests/lib/boot.sh

```diff
@@ -33,3 +33,15 @@ bootenv_unset() {
         fw_setenv "$var"
     fi
 }
+
+get_boot_path() {
+    if [ -d /boot/uboot/ ] && ! [ -z "$(ls -A /boot/uboot/)" ]; then
```

Comment:
Our go-code code uses `/boot/uboot/uboot.env` and `/boot/grub/grubenv` to find the right bootloader, might be nice to do the same here.